src/CMakeLists.txt

```diff
@@ -274,7 +274,6 @@ if(EXIV2_BUILD_EXIV2_COMMAND)
     endif()
 
     install(TARGETS exiv2 RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
-    install( FILES exiv2.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 )
```

Comment:
Eh, why are you stopping the installation of the man page?

Reply:
It is now in the main `CMakeLists.txt`. It looks like we moved things around in master, compared to 0.27-maintenance. But the man page is still being installed 😉